dvc/remote/gdrive.py

```diff
@@ -121,8 +127,12 @@ def __init__(self, repo, config):
         self._service_account_p12_file_path = config.get(
             "gdrive_service_account_p12_file_path"
         )
-        self._client_id = config.get("gdrive_client_id")
-        self._client_secret = config.get("gdrive_client_secret")
+        self._client_id = config.get(
+            "gdrive_client_id", self.DEFAULT_GDRIVE_CLIENT_ID
```

Comment:
need to update here to avoid default value

Reply:
yep, that was left accidentally